src/main/java/pl/isa/biblioteka/BooksEdit.java

```diff
@@ -1,5 +1,35 @@
 package pl.isa.biblioteka;
 
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
 public class BooksEdit {
```

Comment:
Można byłoby to nazwać bardziej w stylu BookCatalog, jako że klasa generalnie jest kolekcją przechowującą książki